gitHubStudent-hometown.md

```diff
@@ -0,0 +1,151 @@
+## Leesville, LA
+
+### Great places to eat
```

Comment:
Was actually just born there so it's technically my 'hometown' don't really know any restaurants down there : )